Makefile

```diff
@@ -97,7 +96,7 @@ endif
 ## Ensures NPM dependencies are installed without having to run this all the time.
 webapp/node_modules: $(wildcard webapp/package.json)
 ifneq ($(HAS_WEBAPP),)
-	cd webapp && $(YARN) install
+	cd webapp && $(NPM) install --ignore-scripts --legacy-peer-deps
```

Comment:
/Not/ copying calls here... yet. Will come back to this later.

Reply:
yah, it's a little complicated. Only do so if you need an updated redux or types. :)